float64 simTime
uint32 playerId
float32 steeringWheel
float32 steeringSpeed
float32 throttlePedal
float32 brakePedal
float32 clutchPedal
float32 accelTgt
float32 steeringTgt
float64 curvatureTgt
float32 steeringTorque
float32 engineTorqueTgt
float32 speedTgt
uint8 gear
uint8 sourceId
uint8[2] spare0
uint32 validityFlags
uint32 flags
uint32 mockupInput0
uint32 mockupInput1
uint32 mockupInput2
uint32 spare